lib/util/markdown.ts

```diff
@@ -5,19 +5,23 @@ import { regEx } from './regex';
 // Generic replacements/link-breakers
 export function sanitizeMarkdown(markdown: string): string {
   let res = markdown;
+
```

Comment:
Please revert changes to this file

Reply:
Who resolves conversations? Is there a "policy"? Conversation issuer for re-review or the one who adresses/fixes stuff?

Reply:
The targeted user 😉